pkg/edged/containers/container_manager.go

```diff
@@ -58,9 +56,8 @@ import (
 	"github.com/kubeedge/kubeedge/pkg/edged/securitycontext"
 )
 
-//Pod details constants
+//Pod constants required by the manager
 const (
-	// Taken from lmctfy https://github.com/google/lmctfy/blob/master/lmctfy/controllers/cpu_controller.cc
```

Comment:
should this be removed ? any issues if we don't mention the reference ?

Reply:
In this push i haven't touched this part. It is been removed, but i plan to push it at later stage of this cleanup series.